bindings/protocols/modbus/index.template.html

```diff
@@ -9,11 +9,11 @@
             specStatus: "ED",
```

Comment:
Of course, I've made a raft of suggestions on the output artifact, rather than on the template which produced that artifact. This once again underscores that such artifacts should not be under source control (a/k/a `git`), or at least that builds should put them in a different directory than the actual source.

Reply:
That might be true but unless someone steps up and involve W3C to restructure all WoT WG repositiries, we need to live with this :(